### PR TITLE
[UK] Add `Beatmapping`

### PR DESCRIPTION
**Pull request by balner**
Beatmapping article fully translated into ukrainian language.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmapping/uk.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Мапінг
+
+**Мапінг**(також бітмапінг) це процес створювання [бітмап](/wiki/Beatmap) в osu! для проходження гравцями.
```
Comment (minusQuantumNeko) [inline wiki/Beatmapping/uk.md:3]:
```suggestion
**Мапінг** (також бітмапінг) - це процес створювання [бітмап](/wiki/Beatmap) в osu! для проходження гравцями.
```

File: wiki/Beatmapping/uk.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Мапінг охоплює вибір пісні, налаштування [таймінгу](/wiki/Client/Beatmap_editor/Timing), розміщення [об'єктів](/wiki/Gameplay/Hit_object), та (вибірково) [скіннінг](/wiki/Skinning) і [сторібординг](/wiki/Storyboard#storyboarding).
+
+Зазвичай маппери намагаються отримати [рейтинговий](/wiki/Beatmap) статус на своїй бітмапі, який отримується за допомогою [процедури рейтингу бітмапи](/wiki/Beatmap_ranking_procedure).
```
Comment (minusQuantumNeko) [inline wiki/Beatmapping/uk.md:6]: так звучить краще
```suggestion
Зазвичай маппери намагаються отримати [рейтинговий](/wiki/Beatmap) статус для своєї бітмапи, який отримується під час [процедури рейтингу бітмапи](/wiki/Beatmap_ranking_procedure).
```

File: wiki/Beatmapping/uk.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+Через те що osu! є ритм грою, пісня яку ви вибираєте для мапінгу повинна мати очевидний та постійний ритм.
+Намагайтесь уникати пісень які мають зміни ритму (або швидкості) і також повільних пісень до того моменту поки ви не станете більш досвідченими в мапінгу.
+
+## Розпочнімо
+
+Для створення нової бітмапи перейдіть в `Edit` (режим редактора) та перетягніть аудіо файл в osu!.
+(Також ви можете додати аудіофайл через папку `Songs`(пісні).)
```
Comment (minusQuantumNeko) [inline wiki/Beatmapping/uk.md:14]:
```suggestion
(Також ви можете додати аудіофайл через папку `Songs` (пісні).)
```

File: wiki/Beatmapping/uk.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+Згодом відкриється сторінка редактора з вашим аудіо файлом.
```
Comment (minusQuantumNeko) [inline wiki/Beatmapping/uk.md:15]:
```suggestion
Після цього відкриється сторінка редактора з вашим аудіо файлом.
```

File: wiki/Beatmapping/uk.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+Ваша нова пісня буде доступна знизу в списку з піснями, позначена фіолетовим кольором.
+Якщо вона не зазначена в виборі пісень за  будь-яких причин, натисніть `F5` щоб скинути кеш бітмап і згодом вона з'явиться.
```
Comment (minusQuantumNeko) [inline wiki/Beatmapping/uk.md:17]:
```suggestion
Якщо вона не зазначена в виборі пісень за будь-яких причин, натисніть `F5` щоб видалити кеш бітмап і згодом вона з'явиться.
```

File: wiki/Beatmapping/uk.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+Натисніть на пісню, після чого відкриється [редактор бітмапи](/wiki/Client/Beatmap_editor) та  вікно `Song Setup`(налаштування пісні).
```
Comment (minusQuantumNeko) [inline wiki/Beatmapping/uk.md:18]:
```suggestion
Натисніть на пісню, після чого відкриється [редактор бітмапи](/wiki/Client/Beatmap_editor) та вікно `Song Setup` (налаштування пісні).
```

File: wiki/Beatmapping/uk.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+### Налаштування пісні
+
+*Головна сторінка: [Налаштування пісні](/wiki/Client/Beatmap_editor/Song_setup)*
+
+При розробці нової [бітмапи](/wiki/Beatmap) ви завжди будете бачити це вікно перед будь-якими іншими частинами редактора.
```
Comment (minusQuantumNeko) [inline wiki/Beatmapping/uk.md:24]:
```suggestion
При створенні нової [бітмапи](/wiki/Beatmap) ви завжди будете бачити це вікно перед будь-якими іншими частинами редактора.
```

File: wiki/Beatmapping/uk.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+### Композиція
+
+*Головна сторінка: [Композиція](/wiki/Client/Beatmap_editor/Compose)*
+
+Вкладка композиція це місце в редакторі в якому ви напевне проведете більшість вашого часу маппінгу.
+Це місце де ви зможете візуально розставляти ігрові об'єкти та вмикати їх [хітсаунди](/wiki/Beatmapping/Hitsound).
+
+### Дизайн
+
+*Головна сторінка: [Дизайн](/wiki/Client/Beatmap_editor/Design) and [Storyboarding](/wiki/Storyboard#storyboarding)*
```
Comment (minusQuantumNeko) [inline wiki/Beatmapping/uk.md:53]:
```suggestion
*Головна сторінка: [Дизайн](/wiki/Client/Beatmap_editor/Design) та [Сторібоардинг](/wiki/Storyboard#storyboarding)*
```

File: wiki/Beatmapping/uk.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+В редакторі вкладка дизайн є місцем де до вашої бітмапи ви можете встановити картинку фону, додати відео і/або зробити звичайний сторіборд.
```
Comment (minusQuantumNeko) [inline wiki/Beatmapping/uk.md:55]:
```suggestion
В редакторі вкладка дизайн є місцем де ви можете встановити картинку фону, додати відео і/або зробити звичайний сторіборд до вашої бітмапи.
```

File: wiki/Beatmapping/uk.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+В залежності від того що ви робите, ви можете використовувати вкладку дизайн як часто (якщо ви займаєтесь [SBS](/wiki/Storyboard/Scripting)(сторіборд скріптинг)) так і взагалі не використовувати.
```
Comment (minusQuantumNeko) [inline wiki/Beatmapping/uk.md:56]:
```suggestion
В залежності від того що ви робите, ви можете використовувати вкладку дизайн як часто (якщо ви займаєтесь [SBS](/wiki/Storyboard/Scripting) (сторіборд скріптингом)) так і взагалі не використовувати.
```

**Review by minusQuantumNeko — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**